livestream/db.go

```diff
@@ -12,7 +12,7 @@ func getPGConn() *pgx.Conn {
 	url := viper.GetString("postgres.url")
 	conn, err := pgx.Connect(context.Background(), url)
 	if err != nil {
-		log.Panicf("Unable to connect to database: %v\n", err)
```

Comment:
`Panicf` would call `panic()`, so I guess now this will return a `nil` `conn` pointer? Not sure which is preferred, since I imagine the `nil` pointer will cause a crash immediately after.

Reply:
Good point – made the error bubble up properly